`/subscriptions` endpoint gives a `404` error

In the test[ `test_id_conversion_bug_v5`](https://github.com/interuss/dss/blob/45a0286a24f35a97715446d9b52f8c05476812b1/monitoring/prober/scd/test_operation_special_cases.py#L157) in the file `test_operation_special_cases.py` , when trying to call the subscription end point an error `404` is received. From the [v0.3.17 `/subscriptions/{sub_id}` endpoint](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/astm-utm/Protocol/master/utm.yaml#operation/createSubscription) it seems like `resp = scd_session.put('/subscriptions/{}'.format(sub_uuid), json=req)` should give one of the prescribed error codes. But it does not. 

```
=================================== FAILURES ===================================
__________________________ test_id_conversion_bug_v5 ___________________________

ids = <function ids.<locals>.<lambda> at 0x7f62130e1310>, scd_api = '0.3.17'
scd_session = <monitoring.monitorlib.infrastructure.DSSTestSession object at 0x7f6213190d90>

    @for_api_versions(scd.API_0_3_5, scd.API_0_3_17)
    @default_scope(SCOPE_SC)
    def test_id_conversion_bug_v5(ids, scd_api, scd_session):
      sub_uuid = ids(SUB_TYPE)
      time_ref = datetime.datetime.utcnow() + datetime.timedelta(days=1)
      time_start = datetime.datetime(time_ref.year, time_ref.month, time_ref.day, 1, 30)
      time_end = datetime.datetime(time_ref.year, time_ref.month, time_ref.day, 22, 15)
      req = {
        "extents": {
          "volume": {
            "outline_polygon": {
              "vertices": [
                { "lng": -91.49723052978516, "lat": 41.70085834502109 },
                { "lng": -91.50341033935547, "lat": 41.6770148220322 },
                { "lng": -91.47989273071289, "lat": 41.67509157220958 },
                { "lng": -91.4663314819336, "lat": 41.69329603398001 }
              ]
            },
            "altitude_upper": {"units": "M", "reference": "W84", "value": 764.79037},
            "altitude_lower": {"units": "M", "reference": "W84", "value": 23.24352}
          },
          "time_start": {"value": time_start.isoformat() + "Z", "format": "RFC3339"},
          "time_end": {"value": time_end.isoformat() + "Z", "format": "RFC3339"}
        },
        "old_version": 0,
        "uss_base_url": "http://localhost:12012/services/uss/public/uss/v1/",
        "notify_for_constraints": True
      }
      resp = scd_session.put('/subscriptions/{}'.format(sub_uuid), json=req)
>     assert resp.status_code == 200, resp.content
E     AssertionError: b'{
E         "error": "Subscription 000000d4-ffff-40ff-a64b-a8fb8db60000 not found",
E         "code": 5,
E         "message": "Subscription 000000d4-ffff-40ff-a64b-a8fb8db60000 not found",
E         "error_id": "E:f8e22b99-ffd5-4959-997a-4e358c317f80"
E       }'
E     assert 404 == 200
E      +  where 404 = <Response [404]>.status_code

scd/test_operation_special_cases.py:189: AssertionError
```

--- a/pkg/scd/subscriptions_handler.go
+++ b/pkg/scd/subscriptions_handler.go
@@ -2,7 +2,6 @@ package scd
 
 import (
 	"context"
-	"time"
 
 	"github.com/golang/geo/s2"
 	"github.com/interuss/dss/pkg/api/v1/scdpb"
@@ -21,8 +20,7 @@ var (
 )
 
 func (a *Server) CreateSubscription(ctx context.Context, req *scdpb.CreateSubscriptionRequest) (*scdpb.PutSubscriptionResponse, error) {
-	version := dssmodels.VersionFromTime(time.Now())
-	return a.PutSubscription(ctx, req.GetSubscriptionid(), version, req.GetParams())
+	return a.PutSubscription(ctx, req.GetSubscriptionid(), nil, req.GetParams())
 }
 
 func (a *Server) UpdateSubscription(ctx context.Context, req *scdpb.UpdateSubscriptionRequest) (*scdpb.PutSubscriptionResponse, error) {
@@ -90,6 +88,8 @@ func (a *Server) PutSubscription(ctx context.Context, subscriptionid string, ver
 	if !subreq.NotifyForOperationalIntents && !subreq.NotifyForConstraints {
 		return nil, stacktrace.NewErrorWithCode(dsserr.BadRequest, "No notification triggers requested for Subscription")
 	}
+
+	// TODO: Check scopes to verify requested information (op intents or constraints) may be requested
 
 	var result *scdpb.PutSubscriptionResponse
 	action := func(ctx context.Context, r repos.Repository) (err error) {

